app/case/apps.tsx

```diff
@@ -16,51 +16,51 @@ const cardsData: cardData[] = [
     content:
       "Discover exclusive discounts on dining, leisure, and entertainment activities.",
     externalLink: "https://www.theentertainerme.com/en-ae/abu-dhabi-al-ain",
-    location: "/img/tips/entertainer.png",
+    location: "/img/apps/entertainer.png",
   },
   {
     title: "Tripadvisor",
     content: "Read reviews and find the best places in Abu Dhabi.",
     externalLink: "https://www.tripadvisor.com",
-    location: "/img/tips/tripadvisor.png",
+    location: "/img/apps/tripadvisor.png",
   },
   {
     title: "Zomato",
     content: "Discover top restaurants and cafes in Abu Dhabi.",
     externalLink: "https://www.zomato.com",
-    location: "/img/tips/zomato.png",
+    location: "/img/apps/zomato.png",
   },
   {
     title: "Careem",
     content: "Book rides quickly and conveniently across the city.",
     externalLink: "https://www.careem.com",
-    location: "/img/tips/careem.png",
+    location: "/img/apps/careem.png",
   },
   {
-    title: "ADDC",
+    title: "TAQA",
     content: "Pay your water and electricity bills quickly and securely.",
     externalLink: "https://www.addc.ae/en-US/home/pages/AboutUs.aspx",
```

Comment:
Lines 41,42 need to be changed in `app/case/apps.tsx`
Please update the content and external link accordingly too